waku/v2/protocol/waku_store/waku_store_types.nim

```diff
@@ -17,6 +17,8 @@ export pagination
 const MaxPageSize* = uint64(100) # Maximum number of waku messages in each page
 
 type
+  HistoryContentFilter* = object
+    topic*: ContentTopic
```

Comment:
I'd suggest we name this now `contentTopic` to avoid confusion as per [this discussion](https://github.com/vacp2p/rfc/issues/327)

Reply:
right! that makes sense! will do.